src/db/knex.js

```diff
@@ -0,0 +1,5 @@
+const env = process.env.NODE_ENV || 'development';
+const knexConfig = require('../../knexfile')[env];
+const knex = require('knex')(knexConfig); // eslint-disable-line import/order
```

Comment:
See what I mean about overriding style? it would be silly to import knex separately without calling the config, so we know better than eslint here and can disable it.